src/app/tests/suites/commands/system/SystemCommands.cpp

```diff
@@ -73,3 +75,32 @@ CHIP_ERROR SystemCommands::RunInternal(const char * command)
     VerifyOrReturnError(system(command) == 0, CHIP_ERROR_INTERNAL);
     return ContinueOnChipMainThread(CHIP_NO_ERROR);
 }
+
+CHIP_ERROR SystemCommands::CreateCommonCommandArgs(char * commandBuffer, size_t commandBufferSize, const char * scriptDir,
+                                                   const char * scriptName, uint16_t discriminator, uint16_t port, const char * kvs)
+{
+    chip::StringBuilder<kCommandMaxLen> builder;
```

Comment:
nit: could have done `StringBuilderBase builder(commandBuffer, commandBufferSize)` and then you do not need an extra strcopy nor extra stack space.

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/17984